jekyll/_cci2/insights-partnerships.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: classic-docs
+title: "Insights Partnerships"
+short-title: "Insights Partnerships"
+version:
+- Cloud
+---
+
+## Overview
+
+{:toc}
+
+This document describes how you can connect Insights data with third party
+providers.
+
+### Sumo Logic integration
+
+Sumo Logic users may track and visualize analytical data across all of their
+jobs on CircleCI. To do so, use the Sumo Logic Orb and Sumo Logic app
+integration from the Sumo Logic partner integrations site.
+
+
+#### The CircleCI dashboard for Sumo Logic
+
+![header]({{ site.baseurl }}/assets/img/docs/CircleCI_SumoLogic_Dashboard.png)
+
+Included panels:
+
+- Total Job
+- Total Successful Jobs
+- Total Failed Jobs
+- Job Outcome
+- Average Runtime in Seconds (by Job)
+- Jobs By Projects
+- Recent Jobs (latest 10)
+- Top 10 Slowest Failed Jobs In Seconds
+- Top 10 Slowest Successful Jobs In Seconds
+
+Install the CircleCI dashboard by using the App Catalog from the dashboard home page.
+
+![header]({{ site.baseurl }}/assets/img/docs/sumologic_app_catalog.png)
+
+This dashboard receives data through the CircleCI Sumo Logic orb which must be included in your projects to be tracked.
+
+#### The Sumo Logic orb
+
+Find the latest version of the Sumo Logic orb on the [Orb Registry](https://circleci.com/developer/orbs/orb/circleci/sumologic).
+
+##### 1. Import the Sumo Logic orb.
+{:.no_toc}
+
+Add the Sumo Logic orb to your project by including the top-level `orbs` key and import `circleci/sumologic@x.y.z` as follows, replacing `x.y.z` with the latest version number at the link above.
+
+```yaml
+orbs:
+  sumologic: circleci/sumologic@x.y.z
+```
+
+##### 2. Add _workflow-collector_ to workflow.
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 52. You can fix by inserting `{: #2-add-workflow-collector-to-workflow }` to the next line